designer/server/plugins/routes/app.ts

```diff
@@ -39,6 +40,43 @@ export const getAppChildRoutes: ServerRoute = {
   },
 };
 
+export const getErrorCrashReport: ServerRoute = {
+  method: "get",
+  path: "/error/crashreport/{id}",
+  options: {
+    handler: async (_request, h) => {
+      try {
+        const { id } = _request.params;
+        const error = _request.yar.get(`error-summary-${id}`) as any;
+        const zip = new JSZip();
+        zip.file(`${id}-crash-report.json`, JSON.stringify(error));
+        const buffer = await zip.generateAsync({
+          type: "nodebuffer",
+          compression: "DEFLATE",
+        });
+        return h
+          .response(buffer)
+          .encoding("binary")
+          .type("application/zip")
+          .header(
+            "content-disposition",
+            `attachment; filename=${id}-crash-report-${new Date().toISOString()}.zip`
```

Comment:
@gstevenson re ${id}, they're the same, so if a form was named at the beginning it would use that but if you select to edit one (instead of new) it will dupe + give it a random ID

Reply:
ok, soudns that could make form maintenance super difficult in future but definitely isn't a problem for the scope of this ticket... maybe in the persistence work

Reply:
in our last meeting haymon suggested ${id}-crash-report for error report file inside the zip , i can follow the same pattern for zip file as well

Reply:
https://collaboration.homeoffice.gov.uk/jira/browse/FORMS-445?focusedCommentId=1396395&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1396395

Jira says:

> Crash report filename should be `<formname>-crash-report.zip`

Let's either have the code match Jira or Jira match the code - they're both different atm